Makefile

```diff
@@ -192,27 +186,48 @@ docker-push: ## Push docker image with the manager.
 	$(CONTAINER_TOOL) push ${IMG}
 
 GO_CONTAINER_CACHE = /tmp/dpu-operator-cache
+REGISTRY ?= $(shell hostname)
+# Use the image urls from the yaml that is used with Kustomize for local
+# development.
+DPU_OPERATOR_IMAGE := $(REGISTRY):5000/dpu-operator:dev
+DPU_DAEMON_IMAGE := $(REGISTRY):5000/dpu-daemon-image:dev
```

Comment:
I like the change, this will make my life a bit easier.

Maybe it would make sense to use ?= here to give users a knob to set DPU_OPERATOR_IMAGE / DPU_DAEMON_IMAGE in the event they are using quay or something instead of a local registry?

Reply:
Aahh disregard, I guess they can do this by adjusting the REGISTRY variable